pkg/v1/tkg/client/validate.go

```diff
@@ -1659,6 +1664,28 @@ func (c *TkgClient) configureAndValidateIPFamilyConfiguration(clusterRole string
 	return nil
 }
 
+func (c *TkgClient) configureAndValidateCoreDNSIP() error {
+	// Core DNS IP is the 10th index of service CIDR subnet
+	// ServiceCIDR must not be empty as it should already been set by configureAndValidateIPFamilyConfiguration if it was omitted
```

Comment:
I think we should always set it since it not meant to be configured.

Reply:
You mean the serviceCIDR? `configureAndValidateIPFamilyConfiguration` that runs before this function will make sure serviceCIDR is not empty.

Reply:
oh misread the comment, I somehow read it as coreDNSIP is set if omitted.